sw/device/silicon_creator/lib/base/sec_mmio.c

```diff
@@ -0,0 +1,123 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/base/sec_mmio.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/silicon_creator/lib/base/abs_mmio.h"
+
+// FIXME: Linker configuration.
+extern sec_mmio_ctx_t sec_mmio_ctx;
+
+// FIXME: Replace for shutdown module handler.
+static sec_mmio_shutdown_handler __shutdown_handler;
```

Comment:
I think I said this in a different version of this CL, but names starting with __ are reserved words in C. I think it's worth picking a different one here...

Reply:
Done. Updated to `sec_mmio_shutdown_cb`.